nifi-extension-bundles/nifi-snowflake-bundle/nifi-snowflake-snowpipe-streaming/src/main/resources/META-INF.services/org.apache.nifi.processor.Processor

```diff
@@ -0,0 +1,15 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+org.apache.nifi.processors.snowflake.snowpipe.streaming.PutSnowpipeStream
```

Comment:
Should be "PutSnowpipeStreaming"

Reply:
And the directory this is in needs to be changed from 'META-INF.services' to 'META-INF/services'